client/src/components/SuggestChanges/SuggestChanges.js

```diff
@@ -14,17 +14,26 @@ export default class SuggestChanges extends React.Component {
     }
 
     render() {
+        const { mode = 'update' } = this.props;
+        const isUpdate = mode === 'update';
         return (
             <Card>
                 <Card.Header>
                     <Card.Title>
-                        Suggest a Change
+                        Suggest a {isUpdate ? 'Change' : 'New Monument or Memorial'}
                     </Card.Title>
                 </Card.Header>
                 <Card.Body>
-                    <p>We pride ourselves on keeping up to date and accurate information.</p>
-                    <p>If you think something on this page is incorrect or outdated, please suggest a change. If your change is approved we'll send you an email to let you know, and you will be listed as a contributor on this page!</p>
-                    <Button variant="primary" onClick={() => this.handleButtonClick()}>SUGGEST A CHANGE</Button>
+                    {isUpdate && <>
+                        <p>We pride ourselves on keeping up to date and accurate information.</p>
+                        <p>If you think something on this page is incorrect or outdated, please suggest a change. If your change is approved we'll send you an email to let you know, and you will be listed as a contributor on this page!</p>
+                        <Button variant="primary" onClick={() => this.handleButtonClick()}>SUGGEST A CHANGE</Button>
+                    </>}
+                    {!isUpdate && <>
+                        <p>Contributions help us to expand our database of Monuments and Memorials.</p>
+                        <p>Your suggestion will be reviewed by our team and then published for the public to see. Your contribution will be credited on the monument or memorial's page.</p>
```

Comment:
I think the wording of the second sentence is a little confusing.

Reply:
I'm open to suggestions. I figured Dr. Decker would have some